server/routes/submissions/getUserContestSubmissions.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express')
+const { verifyUserAccessToken } = require('../middlewares')
+const router = express.Router()
+const { getUserContestSubmissions } = require('../../models/submissions')
+
+router.get(
+  '/:contestId/users/:userId/submissions',
+  verifyUserAccessToken,
+  (req, res) => {
+    getUserContestSubmissions(req)
+      .then((results) => {
+        return res.status(200).json({
+          success: true,
+          results,
+          error: null,
+        })
+      })
```

Comment:
Question: CSV won't be required for this?....if user wants to keep his record

Reply:
CSV would be generally used by admins and moderators mainly for grading purpose. But, it doesn't seem useful for individual users.